### PR TITLE
Avoid WPI crash when processing records

### PR DESCRIPTION
**Pull request by kelloggm**

### CODE REVIEW

File: checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java [hunk 1]
```diff
@@ -26,6 +26,7 @@ public AinferTestCheckerJaifsTest(List<File> testFiles) {
         "ainfer-testchecker/non-annotated",
         "-Anomsgtext",
         "-Ainfer=jaifs",
+        "-AskipDefs=SimpleRecord",
```
Comment (mernst) [inline checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java:29]: What is the reason for `-AskipDefs` here and below?  It seems that would avoid processing the new test case that you added.

**Review by mernst — CHANGES_REQUESTED** [review]

File: checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java (hunk 1 above)
Comment (kelloggm) [reply checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java:29]: The AFU's jaif reading/writing code doesn't support Java 17, and so executing jaif-based WPI on this test case causes a crash in the AFU.

Comment (kelloggm) [reply checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/AinferTestCheckerJaifsTest.java:29]: I'll add a comment to that effect here.

**Review by mernst — APPROVED** [review]
Thanks!

**Pull request merged**